### PR TITLE
Issue 87 guessCast

### PR DESCRIPTION
**Pull request by spgarbet**
Not completely happy with where this landed, but it's close and it works. Open to design changes.  #87

### CODE REVIEW

File: R/guessCast.R
Comment (nutterb) [inline R/guessCast.R:1]: 1. Rename `conn` to `rcon` for consistency with our other methods.
2. The example starting on line 22 won't run. It is missing the `rcon` argument.
3. There are no argument validations in either `guessCast` nor `guessDate`. You like living dangerously? :)

That's all I see from scanning the code. I'll be able to load and play with it over the weekend.

Comment (spgarbet) [reply R/guessCast.R:1]: I'll fix all those. I'm coding on covid it's not going well. :)

Comment (nutterb) [reply R/guessCast.R:1]: Your "not going well" looks an awful lot like my A-game. consider me impressed.

**Comment by spgarbet** [comment]
Creating those parameter checks and more tests uncovered a nasty logic bug in threshold triggering. It was well worth the effort.

File: R/guessCast.R
Comment (spgarbet) [reply R/guessCast.R:1]: All done. It uncovered some bugs.

**Review by nutterb — APPROVED** [review]
I get four failing tests, but those are related to changes in testing made in the purge/restore branch and can be ignored (the purge-restore tests are more robust). 

Check passes cleanly.

**Pull request merged**